tests/main/views/test_process_outcomes.py

```diff
@@ -0,0 +1,918 @@
+                        "awardValue": "271271.20",
+                        "startDate": "2020-10-10",
+                        "endDate": "2020-11-20",
+                    },
+                }),
+            },
+        ),
+        (
+            # other_po_data
+            None,
+            # initial_data
+            {
+                "completed_at": None,
+                "awarding_organisation_name": "Lambay Freehold",
+                "award_value": 54321,
+                "start_date": datetime.date(2010, 12, 12),
+                "end_date": datetime.date(2011, 12, 12),
+            },
+            # put_values
+            {
+                "completed": False,
+                "award": {
+                    "startDate": None,
+                },
+            },
+            # expected_status_code
+            200,
+            # expected_response_data
+            {
+                "processOutcome": AnySupersetOf({
+                    "completed": False,
+                    "award": {
+                        "awardingOrganisationName": "Lambay Freehold",
+                        "awardValue": "54321.00",
+                        "startDate": None,
+                        "endDate": "2011-12-12",
+                    },
+                }),
+            },
+        ),
+        (
+            # other_po_data
+            {},
+            # initial_data
+            {
+                "completed_at": None,
+                "awarding_organisation_name": "Lambay Freehold",
+                "award_value": 5432.1,
+                "start_date": datetime.date(2010, 12, 12),
+                "end_date": datetime.date(2011, 12, 12),
+            },
+            # put_values
+            {
+                "completed": True,
+            },
+            # expected_status_code
+            200,
+            # expected_response_data
+            {
+                "processOutcome": AnySupersetOf({
+                    "completed": True,
+                    "award": {
+                        "awardingOrganisationName": "Lambay Freehold",
+                        "awardValue": "5432.10",
+                        "startDate": "2010-12-12",
+                        "endDate": "2011-12-12",
+                    },
+                }),
+            },
+        ),
+        (
+            # other_po_data
+            {},
+            # initial_data
+            {
+                "completed_at": None,
+                "result": "none-suitable",
+            },
+            # put_values
+            {
+                "award": {
+                    "awardingOrganisationName": "Talbot de Malahide",
+                },
+            },
+            # expected_status_code
+            400,
+            # expected_response_data
+            {
+                "error": (
+                    "awarding_organisation_name cannot be set for 'none-suitable' "
+                    + "ProcessOutcomes. Attempted to set value 'Talbot de Malahide'"
```

Comment:
The plus isn't necessary

Reply:
Just there as an obvious indicator that it's not a sequence of separate strings - commas are easy to miss.

Reply:
I thought you didn't need anything, just whitespace, but doesn't matter

Reply:
Oh no, you don't need anything. It's just that the only thing that tells you that it's *not* two separate strings is the *lack of* a little comma, so it's a stylistic quirk I sometimes do, but I'm not strongly attached to it.